thunderfish/eodanalysis.py

```diff
@@ -1,7 +1,8 @@
 import numpy as np
 import sys
+import peakdetection as pkd
 
-def eod_extracting(bwin_data, samplerate, fish_type, psd_type):
+def eod_extracting(bwin_data, samplerate, fish_type, psd_type, pulse_window = 0.006):
```

Comment:
Make this independent of the fish_type (and psd_type). Only supply an appropriate data window length.